server/cmd/mmctl/commands/enterprise.go

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+//go:build enterprise
+// +build enterprise
+
```

Comment:
Let's add a comment mentioning the JIRA ticket and warn people against deleting the file.

Reply:
Good suggestion, added :+1: